ogc-example/src/main/scala/geotrellis/server/ogc/conf/OgcSourceConf.scala

```diff
@@ -17,37 +17,37 @@
 package geotrellis.server.ogc.conf
 
 import geotrellis.server.ogc._
+import geotrellis.raster.{RasterSource, ResampleMethod}
 
-import geotrellis.raster.RasterSource
 import com.azavea.maml.ast._
-import cats._
-import cats.implicits._
-
 
 // This sumtype corresponds to the in-config representation of a source
 sealed trait OgcSourceConf {
   def name: String
   def styles: List[StyleConf]
+  def resampleMethod: ResampleMethod
 }
 
 case class SimpleSourceConf(
   name: String,
   title: String,
   sources: List[String],
   defaultStyle: Option[String],
-  styles: List[StyleConf]
+  styles: List[StyleConf],
+  resampleMethod: ResampleMethod
```

Comment:
Should this default instead to "nearest-neighbor" so that existing confs are backwards compatible rather than forcing each user to make a choice about each layer they've defined? A default would also reduce the burden on new users when generating their first conf.

I don't feel strongly about it, just thinking through the implications of a change like this. If we leave it as is, we should definitely clarify in the changelog that this change is breaking for existing confs and point users to the list of available options and even perhaps what the implications of each option are, if we have that info available.

Reply:
I also didn't feel strong; see https://github.com/geotrellis/geotrellis-server/pull/233#discussion_r396420013 :D I'll add a default behavior than, since I also had a feeling about that but was not sure :+1:

Reply:
Oh I missed that comment -- :great-minds:. Adding a default sounds good, just leave the non-default new layer in the application.conf to demonstrate that it's possible. Maybe link to the non-default layer in the changelog so users can see an example at a glance?

Reply:
@CloudNiner :+1: